mParticle-OneTrust.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
     s.social_media_url = "https://twitter.com/mparticle"
 
     
-    s.ios.deployment_target = "9.0"
+    s.ios.deployment_target = "11.0"
     s.ios.source_files      = 'mParticle-OneTrust/*.{h,m}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 8.0'
     s.ios.dependency 'OneTrust-CMP-XCFramework', '~> 6.17'
```

Comment:
@alexs-mparticle I don't see any methods from the OneTrust SDK being called in any of the changes in this PR, so it might not be useful to have this included. We update our version every three weeks, so if we include this in the podspec, I think that it will limit the version of OneTrust clients can install.

Reply:
This is just the minimum version - and uses wildcards up to (but not including) 7.0. it's still on app developers to choose which version in their own podfile.

Reply:
So we're on version 6.19.0 right now, and it's likely that we'll roll out a version 7 within the next year. Perhaps it doesn't need to be adjusted right now, but it's something to keep an eye on.